Formula/vsftpd.rb

```diff
@@ -49,6 +49,31 @@ def caveats
 
     s
   end
+
+  plist_options :startup => true, :manual => "sudo #{HOMEBREW_PREFIX}/sbin/vsftpd #{HOMEBREW_PREFIX}/etc/vsftpd.conf"
```

Comment:
Thanks. I will update the pull request. In fact, the manual command can be just `sudo vsftpd`.